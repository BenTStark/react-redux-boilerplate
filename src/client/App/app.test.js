import React from 'react'
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer'
import AppComponent from "./app.component";

// Snapshot for Home React Component
describe(">>> AppComponent - Snapshot", () => {
   it("+++capturing Snapshot of AppComponent", () => {
  //   const renderedValue = renderer.create(<AppComponent />).toJSON();
  //   expect(renderedValue).toMatchSnapshot();
  });
});
