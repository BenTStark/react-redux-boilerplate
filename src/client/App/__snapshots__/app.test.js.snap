// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> AppComponent - Snapshot +++capturing Snapshot of AppComponent 1`] = `
<div
  className="app"
>
  <Connect(NavigationComponent) />
  <div>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        path="/otherRoute"
        render={[Function]}
      />
      <Route
        component={[Function]}
        path="/blog"
      />
      <Route
        component={[Function]}
        path="/contact"
      />
      <Route
        path="/callback"
        render={[Function]}
      />
      <Route
        component={[Function]}
        status={404}
      />
    </Switch>
  </div>
  <div>
    Footer
  </div>
</div>
`;
